test/intl402/Segmenter/constructor/constructor/options-granularity-abrupt-throws.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.segmenter
+description:
+    Return abrupt completion from GetOption localeMatcher
```

Comment:
The file name indicates that this should be `granularity`

Reply:
> The file name indicates that this should be `granularity`

sorry. My bad. Must be too tired last night and forget to change it after copy the file.